tests/Doctrine/ODM/MongoDB/Tests/BaseTest.php

```
@@ -144,6 +144,20 @@ protected function skipTestIfNotSharded($className)
         $this->markTestSkipped('Test skipped because server does not support sharding');
     }
 
+    protected function skipTestIfSharded($className)
+    {
+        $result = $this->dm->getDocumentDatabase($className)->command(['listCommands' => true])->toArray()[0];
+        if (! $result['ok']) {
+            $this->markTestSkipped('Could not check whether server supports sharding');
```

Shouldn't this one be marked as `risky` as we can't determine if the test should run or not? This will be easier to spot than `S` which already is showing up in the results.

---

I can also throw an exception to let the test fail: if we can't run `listCommands` this warrants further investigation.

---

Sound good 👍

---

I've removed the check entirely: the `command` method already throws an exception for failed commands, so the checks are total unnecessary. I've removed them.